### PR TITLE
Update mysql-wordpress-persistent-volume.md

### PR DESCRIPTION
**Pull request by mudit000**
corrected switch for kubectl apply and delete from -k to -f

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by mudit000** [comment]
I have signed the CLA

File: content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ The following manifest describes a single-instance MySQL Deployment. The MySQL c
 The `kustomization.yaml` contains all the resources for deploying a WordPress site and a 
 MySQL database. You can apply the directory by
 ```shell
-kubectl apply -k ./
+kubectl apply -f ./
```
Comment (kbhawkey) [inline content/en/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md]: Hello @mudit000. To better understand the change, can you explain why the `-k flag` should be reverted? Are you using `kubectl v1.14`?

**Comment by mudit000** [comment]
Hi @kbhawkey,
I am using 1.12,1.13 and 1.14 at the moment.There is no -k flag.
-f flag is used to apply files(yaml) to create the resource and not -k.
Thanks,
Mudit

**Comment by kbhawkey** [comment]
Hi, @mudit000. I have used `kubectl 1.14` to evaluate this tutorial. If I recall correctly, there is a `-k` flag available on the `apply` command for kubectl v1.14. 
See issue #13814.

**Comment by mudit000** [comment]
Hi @kbhawkey ,
I need to check this on 1.14.Currently i faced this issue on 1.12 .
Thanks,
Mudit

**Comment by kbhawkey** [comment]
@mudit000, the -k flag was introduced in version 1.14 .
I also could not find the -k flag in the kubectl reference, https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply. I will log an issue.
The `kubectl` book does refer to the new flag, https://kubectl.docs.kubernetes.io/pages/app_management/apply.html

**Comment by mudit000** [comment]
@kbhawkey:I was able to get further details of flag -k by reading the release notes.Thanks!

**Comment by DanyC97** [comment]
@mudit000 i saw that you close the PR and then reopened, was that the intention ? if so please provide some info as to why you decided to reopen it, thank you

**Comment by mudit000** [comment]
Hi @kbhawkey 
How can i raise pull request for #13961 or do i need to.?

**Comment by kbhawkey** [comment]
> 
> 
> Hi @kbhawkey
> How can i raise pull request for #13961 or do i need to.?

Hi, @mudit000. Before opening a new PR, I would investigate whether docs were generated incorporating the latest 1.1.4 `kubectl` changes. I pinged the 1.1.4 docs release mgr. The `kubectl` help includes the latest command/flag changes.

**Comment by jaredbhatti** [comment]
hey there - any update on this?

**Comment by jaredbhatti** [comment]
It looks like -f is correct: https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

**Comment by kbhawkey** [comment]
Hello @jaredbhatti, You can use apply -k (with directory containing kustomization.yaml) or apply -f (with file). I created a new PR with the latest kubectl ref generated, #14018. I don't think the existing kubectl ref docs are current for 1.14, but not totally sure. The -k flag was introduced in 1.14.

**Comment by mudit000** [comment]
Hi @kbhawkey,
Let me know if i can close this PR.
Thanks,
Mudit

**Comment by mudit000** [comment]
> @mudit000 i saw that you close the PR and then reopened, was that the intention ? if so please provide some info as to why you decided to reopen it, thank you
Hi @DanyC97,That was by mistake.

Thanks,
Mudit

**Pull request closed without merging**